conf/bafu.conf.part

```diff
@@ -17,6 +17,91 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
         from hydrologie.hydrologie_untersuchungsgebiete_stationen
 }
 
+source src_ch_bafu_vec25-gewaessernetz_2000 : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+        , name as label \
+        , 'feature' as origin \
+        , remove_accents(bgdi_id::text||' '||gwlnr||' '||gewissnr::text||' '||name||' '||objectval) as detail \
```

Comment:
bgdi_id should not be used as feature id (linkt to object?)  and should not be searchable, it is not stable.